ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -871,7 +871,8 @@ then
 			if ver hwcmp PX4FMU_V5
 			then
 				set LOGGER_BUF 64
-				param set SDLOG_MODE 3
+				param set SDLOG_MODE 2
+				param set SDLOG_PROFILE 2
```

Comment:
Why is this set to thermal calibration? It comes from 68e76d8ed3a19 and I don't think that was the intention.